### PR TITLE
feat: New --include and --exclude parameters

### PR DESCRIPTION
**Pull request by vinigfer**
This will enable to use this action with more scenarios

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,6 +14,16 @@ This tells `black` how long each line of Python code can be.
...
 
+## Include (optional)
+This tells `black` which files we should verify the format.
+
+**Default**: `\.pyi?$`, which is `black`'s default value if not informed.
+
+## Exclude (optional)
+This tells `black` which files we should ignore.
+
+**Default**: `(\.direnv|\.eggs|\.git|\.hg|\.mypy_cache|\.nox|\.tox|\.venv|\.svn|_build|buck-out|build|dist)`, which is `black`'s default value if not informed.
```
Comment (rodrigogiraoserrao) [inline README.md]: Doesn't match the default value [in the black repository](https://github.com/psf/black#command-line-options), as it's missing the leading and trailing `/`.

File: action.yaml [hunk 2]
```diff
@@ -13,6 +13,14 @@ inputs:
...
     required: false
     default: '81'
+  include:
+    description: 'A regular expression that matches files and directories that should be included on recursive searches.'
+    required: false
+    default: '\.pyi?$'
+  exclude:
+    description: 'A regular expression that matches files and directories that should be excluded on recursive searches.'
+    required: false
+    default: '(\.direnv|\.eggs|\.git|\.hg|\.mypy_cache|\.nox|\.tox|\.venv|\.svn|_build|buck-out|build|dist)'
```
Comment (rodrigogiraoserrao) [inline action.yaml]: Again, this should match the actual `black` default value.

**Review by rodrigogiraoserrao — CHANGES_REQUESTED** [review]
Thanks for the PR; as soon as you fix the issues with the comments I made I'll be happy to merge this.

File: README.md (hunk 1 above)
Comment (vinigfer) [reply README.md]: Weird ... tried yesterday with the trailling and ending slash, but did not worked. That's why I removed.
Anyway, tried again now and it seems to be working. Have a look please

**Review by rodrigogiraoserrao — APPROVED** [review]
Thanks for the PR!

**Pull request merged**